_data/sessions.yml

```
@@ -4,63 +4,194 @@
+  subtype: presentation
+  speakers: [207]
+
+-
+  id: 208
+  title: "Experiences from 'High-performance R' course "
+  description: "A researcher using R to analyse their data and conduct statistical analyses may run into a situation where the computational limits of their laptop have been reached or the analyses simply take too long. What to do then? Depending on the researcher's background, R is approached either as a programming language or as an off-the-shelf statistical analysis software via RStudio. We set out to develop an R course in CSC that aims at teaching how R can be used efficiently by looking at how R is designed as a programming language to facilitate analyses in an efficient way in terms of hardware resources and then extending to R usage in an HPC environment. The course is aimed for any researcher using R as their main programming language and wanting to expand their knowledge of R. The instructors' backgrounds include both computer science and a field-specific data analysis perspective in using R. In this talk we give an overview of the course content, discuss experiences from the first course instance in autumn 2024, and outline some future plans."
+  subtype: presentation
+  speakers: [208]
+
+-
+  id: 209
+  title: "N-min-ct - a research software to tell how sustainable is the neighbourhood."
+  description: "Cities of the future are challenged to cope with population growth and an increasing demand for services while reducing their envronmental footprints. A part of a solution is in designing better connected road networks and in transforming urban areas such that minimise commuting time. Hence, Trondheim is among the first to apply the principles of a 15-minute-city - most destinations needed on daily basis are to be found within 15 minutes of commuting time. Analysis of quantity and quality of urban services within reach, given the origin of the trip, various means of transport, existing or planned road networks and other variables, however, is a computationally challenging task that needs custom software. This talk presents an ongoing development of a web-instrument to meet such needs in research on urban planning, and elaborates on desig principles of applications based on graph theory, geospatial computing, complex data pipelines and GPU-acceleration."
+  subtype: presentation
+  speakers: [209]
+
+-
+  id: 301
+  title: "Do you have a moment to talk about our lord and savior, vim?"
+  description: "For decades have I resisted the idea of learning its infamously arcane and peculiar set of shortcuts. Last year I decided to face my fears, and was amazed at how quickly I learned the basics and how useful they have become in nearly all aspects of my digital life. In this tutorial, I will share the Good News; we'll go through some basic operations and talk a bit about extensions and configurations. Experienced users are also welcome to join and share their favorite tips."
+  subtype: tutorial
+  speakers: []
```

No speakers yet in the vim session.

---

```suggestion
  speakers: [301]
```

luckily I numbered things logically so that ids match